### PR TITLE
refactor fluentd-cloudwatch

### PR DESCRIPTION
**Pull request by cheyang**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/fluentd-cloudwatch/README.md [hunk 1]
```diff
@@ -46,9 +46,12 @@ The following tables lists the configurable parameters of the Fluentd Cloudwatch
...
 | Parameter                       | Description                                | Default                                                    |
 | ------------------------------- | ------------------------------------------ | ---------------------------------------------------------- |
-| `image`                         | Image                                      | `fluent/fluentd-kubernetes-daemonset`                      |
-| `imageTag`                      | Image tag                                  | `v0.12.33-cloudwatch`                                      |
-| `imagePullPolicy`               | Image pull policy                          | `Always` if `imageTag` is `imagePullPolicy`                |
+| `image.repository`              | Image repository                           | `fluent/fluentd-kubernetes-daemonset`            
+|
+| `image.tag`                     | Image tag                                  | `v0.12.33-cloudwatch`                                      
+|
+| `image.pullPolicy`              | Image pull policy                          | `Always` if `imageTag` is `image.pullPolicy`
```
Comment (unguiculus) [inline incubator/fluentd-cloudwatch/README.md]: Default is `IfNotPresent`

File: incubator/fluentd-cloudwatch/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 name: fluentd-cloudwatch
-version: 0.2.2
+version: 0.3.0
 appVersion: 0.1.1
```
Comment (unguiculus) [inline incubator/fluentd-cloudwatch/Chart.yaml]: Could you also fix the appVersion which is v0.12.33-cloudwatch

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by unguiculus — APPROVED** [review]

**Comment by cheyang** [comment]
@unguiculus , the reason of failure is that the docker image of fluentd in testing machine is not update to date.
```
I0514 12:28:35.914]   9m		7m		5	kubelet, gke-jenkins-default-pool-51cafeba-czr7	spec.containers{fluentd-9762-1-fluentd-cloudwatch}	Normal		Pulled			Container image "fluent/fluentd-kubernetes-daemonset:v0.12.33-cloudwatch" already present on machine
```

Its SHA is `sha256:0a6763c174ac9456ae3b71ae4485ff5f9ab7ecd5a1542e71248a72c54666f02c`, while the latest docker image with the same tag is `9c467e632224d5060218362d5efc585477870272ebb803fe2341d3261d65c58d`. I run the test in my local machine, the test can be passed.  So what's your idea? Cleaning the docker images in test machine or Just changing pull policy to Always?

**Comment by cheyang** [comment]
``` 
---Logs from container fluentd-9768-1-fluentd-cloudwatch in pod fluentd-9768-1-fluentd-cloudwatch-tn5zs:---
I0514 15:05:14.332] sed: can't create temp file '/fluentd/etc/fluent.confXXXXXX': Read-only file system
I0514 15:05:14.337] ---End of logs for container fluentd-9768-1-fluentd-cloudwatch in pod fluentd-9768-1-fluentd-cloudwatch-tn5zs---
```

I'm not able to reproduce it in my test environment. @unguiculus Do you have any ideas?

**Comment by unguiculus** [comment]
It looks like it is due to configmap mounts being readonly from Kubernetes 1.9.4. What's you cluster version?

**Comment by cheyang** [comment]
My cluster version is 1.9.3.

**Pull request merged**